docs/arf.md

```diff
@@ -772,6 +758,23 @@ established by [Article 5a](https://eur-lex.europa.eu/legal-content/EN/ALL/?uri=
     (QES) Remote Service Provider. This interface is specifically used
     to execute a QES remote signature.
 
+Furthermore, the following interfaces are subject to harmonization, but are
```

Comment:
I'm not sure I understand the difference between harmonization and standardization. I would say those interfaces could be listed to establish a certain terminology. The design is at the discretion of the wallet provider.